flow360/component/flow360_params/flow360_output.py

```diff
@@ -477,10 +523,85 @@ def schema_extra(schema, model):
     def to_solver(self, params, **kwargs) -> MonitorOutput:
         solver_model = super().to_solver(params, **kwargs)
         solver_values = solver_model.__dict__
+        _deduplicate_output_fields(solver_values, "monitors")
         _distribute_shared_output_fields(solver_values, "monitors")
         return MonitorOutput(**solver_values)
 
 
+class NoTypeMonitor(MonitorBase):
```

Comment:
What is means "NoTypeMonitor" ?

Reply:
This it the legacy way of specifying monitor back in the days when we only have probe monitor. So there is no "type" needed.

Reply:
Renamed to `LegacyMonitor`